corehq/apps/receiverwrapper/models.py

```diff
@@ -337,6 +353,27 @@ def payload_doc(self, repeat_record):
         return None
 
 
+@register_repeater_type
+class JsonFormRepeater(FormRepeater):
```

Comment:
What's the purpose of the `JsonFormRepeater`, it doesn't seem to do anything that the normal `FormRepeater` does. Can't you just register the `FormRepeaterDhis2EventPayloadGenerator` to the FormRepeater class?